docs/journal/01-cold-storage.md

```diff
@@ -93,44 +108,47 @@ In this configuration, devices post event data to a Service Bus topic. Worker ro
 
 - **Connectivity.** Devices need to be able to communicate with Service Bus in order to post messages. As before, local field gateways may be necessary to provide this connectivity.
 
-![cloud service solution based on Service Bus Topics and Work Roles](media/01-cold-storage/physical-architecture-worker-role-and-service-bus.png)
 
 - *Other Concerns - TBD*
 
-### Use Azure Event Hub to ingest events and a worker role to process them
+
+### Event Hub + worker role
 
 [Azure Event Hub][event-hubs] is a cloud-scale telemetry ingestion service. It is designed to capture millions of events per second in near real-time.
 
+In this approach, devices (or a field gateway) send events to Event Hub. Worker role instances process the events.
+
+![cloud service solution based on Event Hub and Work Roles](media/01-cold-storage/physical-architecture-worker-role-and-event-hub.png)
+
 **Pros:**
 
 - **Reliability.** As described in the previous approach, the AMQP protocol provides reliability guarantees, helping to ensure that event data is not lost prior to ingestion. Data received by Event Hub is retained for a specified period (which can be measured in days). If a worker role fails when processing the data for an event, it can be restarted and data is not lost.
 
-- **Scalability.** Event Hub is highly scalable through partitioning. An event hub can contain up to 32 partitions; each partition can receive messages in parallel with other partitions. Event Hub is designed to handle a continuing, large influx of events and is capable of processing up to 1MB/second of data per partition; this is well beyond the current requirements of the system that Fabrikam are proposing and allows the same architecture to be used for much larger systems in the future.
+- **Scalability.** Event Hub is highly scalable through partitioning. An event hub can contain up to 32 partitions; each partition can receive messages in parallel with other partitions. Event Hub is designed to handle a continuous, large influx of events. It can process up to 1MB/second of data per partition &mdash; well beyond requirements of our Fabrikam scenario.
 
 - **Security.** You can set security policies to help protect Event Hub and authenticate message senders. AMQP provides transport-level security to protect messages in-flight. Event Hub also supports blacklisting of devices; if a device is compromised or stolen, data transmitted by the device can be blocked on receipt by Event Hub.
 
 **Concerns:**
```

Comment:
> leading to the same scalability concerns described in the previous approach

I would rather say "leading to _one of the_ scalability concerns described in the previous approach"